PRS1001-2000/PRS1015Abarra.xml

```diff
@@ -46,8 +46,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <listPerson>
             <person sex="1">
                <persName>
```

Comment:
please remove the outer persName.
you should have person, and inside person a series of persNames